standalone-metastore/metastore-server/src/main/sql/mssql/hive-schema-4.0.0.mssql.sql

```diff
@@ -101,7 +101,7 @@ CREATE TABLE PART_COL_STATS
 
 ALTER TABLE PART_COL_STATS ADD CONSTRAINT PART_COL_STATS_PK PRIMARY KEY (CS_ID);
 
-CREATE INDEX PCS_STATS_IDX ON PART_COL_STATS (CAT_NAME, DB_NAME,TABLE_NAME,COLUMN_NAME,PARTITION_NAME);
+CREATE INDEX PCS_STATS_IDX ON PART_COL_STATS (DB_NAME,TABLE_NAME,COLUMN_NAME,PARTITION_NAME, CAT_NAME);
```

Comment:
Ideal way would be have the column with highest cardinality come in the front of the composite key. 

(PARTITION_NAME, COLUMN_NAME, TABLE_NAME, DB_NAME, CAT_NAME) would be the better sequence for this index to be used efficiently.

Reply:
@rahulchugh We would like to understand this guidance a bit better. We can understand how columns with high cardinality have better read performance but we are not sure if this would have regressed write performance or at the least have bloated size on storage.  
Can you please point us to some docs/best practices guide on this recommendation? Or if you are familiar with how such indices are stored, could you please educate us? Thank you

Reply:
order doesn't matter if you use '='